ssh-login-alerts/setup-telegram-notification.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Set your Telegram bot token and chat ID
+TOKEN="123456789:ABCDEFGHIJK-ABCDEFGHIJK"
```

Comment:
I would change it to fetch/get/read users input for `TOKEN` and `ID`:

Most portable version would be:
```bash
printf "%s" "Enter TOKEN: "
read TOKEN
```